Ubuntu/master.sh

```diff
@@ -59,9 +59,25 @@ setenforce 0
 sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
 systemctl disable firewalld
 systemctl status firewalld
-rm /etc/containerd/config.toml
+
+mkdir -p /etc/containerd
+containerd config default | tee /etc/containerd/config.toml > /dev/null
+sed -i 's/SystemdCgroup = false/SystemdCgroup = true/' /etc/containerd/config.toml
```

Comment:
Should this be conditional on just 24.04 or shall we standardize on containerd going forward? cc: @aditya-lb @arpit-lb

Reply:
@madhura-lb as you pointed out we are already on containerd with the migration to 1.28. Can you add notes on why previously we were removing the config.toml and now generating a new config with the cgroup setting?

Reply:
We were previously removing config.toml for versions <= 1.23 as it would conflict with Docker. We no longer need to remove it with the recent versions.

We are now creating a new config to ask containerd to use cgroup driver instead of cgroupfs. Kubelet uses cgroup driver to manage container resources. Now if containerd tried to manage that resource using cgroupfs , it will fail to do it.

Without alignment:
- kubelet tries to manage Pod "A" with systemd cgroups
- containerd tries to manage same Pod "A" with cgroupfs
- Result: conflicting resource controls